benchmarks/helics/PholdFederate.hpp

```diff
@@ -9,37 +9,39 @@ SPDX-License-Identifier: BSD-3-Clause
 #include "helics/application_api/MessageFederate.hpp"
 #include "helics/core/ActionMessage.hpp"
 #include "helics/helics-config.h"
+
 #include <random>
 
 using namespace helics;
 /** class implementing a federate for the PHOLD benchmark*/
 class PholdFederate
 {
   public:
-    helics::Time deltaTime = helics::Time (10, time_units::ns);  // sampling rate
-    helics::Time finalTime = helics::Time (10000, time_units::ns);  // final time
-    int evCount = 0; // number of events handled by this federate
+    helics::Time deltaTime = helics::Time(10, time_units::ns);  // sampling rate
+    helics::Time finalTime = helics::Time(10000, time_units::ns);  // final time
+    int evCount            = 0;  // number of events handled by this federate
 
   private:
     std::unique_ptr<helics::MessageFederate> mFed;
-    helics::Endpoint* ept;
+    helics::Endpoint * ept;
```

Comment:
Is this the preferred alignment of pointer indicators?

Reply:
I prefer the new formatted way of aligning the `*`, i.e. spaces before and after the `*`, but I don't feel strongly about this.

Reply:
I slightly prefer not having a space on both sides -- either `helics::Endpoint* ept` or `helics::Endpoint *ept`.

Reply:
If I had to choose one, I'd write `helics::Endpoint* ept` instead of `helics::Endpoint *ept`.